packages/google_maps_flutter/google_maps_flutter_platform_interface/lib/src/types/point_of_interest.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'types.dart';
+
+/// A pair of latitude and longitude coordinates, stored as degrees.
+class PointOfInterest {
+  /// Creates a PointOfInterest
+  PointOfInterest(this.position, this.name, this.placeId);
+
+  /// The LatLng of the POI.
+  final LatLng position;
+
+  /// The name of the POI.
+  String? name;
+
+  /// The placeId of the POI.
```

Comment:
Defining a variable's meaning by repeating its name doesn't provide any information; this should describe it in English.

Reply:
I have provided more information in the comment:
/// The human-readable name of the POI as displayed on the map.